README.md

```diff
@@ -103,3 +103,33 @@ An Awesome List of Celestia Resources
 * [Telegram](https://t.me/CelestiaBrazil)
 
 ## Node operator contributions
+
+## [ChainTrails](https://chaintrails.io)
+* [Documentation](https://docs-celestia.chaintrails.io/)
+* [Ansible Playbooks](https://github.com/chaintrails/celestia-validator-playbooks)
+### Services
```

Comment:
please increment other headers according to the first one with ###

thanks! please also rebase this, as I don't have permission to edit